src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+for (let i = 0; i < li.length; i++) {
```

Comment:
The querySelectorAll method returns a NodeList, which has a .forEach method that is usually preferable to using a for loop. This is helpful because it avoids issues with scope and variable declaration.

Reply:
You can even simpify this function to be just:

```
items.forEach(item => {
  const span = document.createElement('span');

  item.prepend(span);
  span.append(span.nextSibling);
});
```